Kentor.AuthServices/SAML2P/Saml2Response.cs

```diff
@@ -208,6 +208,34 @@ private void ReadAndValidateInResponseTo(XmlElement xml, Saml2Id expectedInRespo
             Saml2Id inResponseTo,
             string relayState,
             Uri audience,
+            params ClaimsIdentity[] claimsIdentities )
+            : this( issuer, issuerCertificate, destinationUrl, inResponseTo, relayState, audience, 
+                null, claimsIdentities )
+        { }
+
+        /// <summary>
+        /// Create a response with the supplied data.
+        /// </summary>
+        /// <param name="issuer">Issuer of the response.</param>
+        /// <param name="issuerCertificate">The certificate to use when signing
+        /// this response in XML form.</param>
+        /// <param name="destinationUrl">The destination Uri for the message</param>
+        /// <param name="inResponseTo">In response to id</param>
+        /// <param name="relayState">RelayState associated with the message.</param>
+        /// <param name="sessionNotOnOrAfter">The time instant at which the session between the principal 
+        /// identified by the subject and the SAML authority issuing this statement
+        /// must be considered ended.</param>
+        /// <param name="claimsIdentities">Claims identities to be included in the 
+        /// <param name="audience">Audience of the response, set as AudienceRestriction</param>
+        /// response. Each identity is translated into a separate assertion.</param>
+        public Saml2Response(
+            EntityId issuer,
+            X509Certificate2 issuerCertificate,
+            Uri destinationUrl,
+            Saml2Id inResponseTo,
+            string relayState,
+            Uri audience,
+            DateTime? sessionNotOnOrAfter,
```

Comment:
Same as previous comment, add a new overload instead of altering the signature of an existing public method.

Reply:
@AndersAbel I believe I did here, is that not what your seeing?